<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeBook</class>
 <widget class="QDialog" name="ChangeBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Автор</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Жанр</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="savechanges">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Отправить изменения</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="date">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
